src/player/player.ts

```diff
@@ -714,5 +716,33 @@ module Shumway.Player {
       });
       symbol.resolveAssetPromise.then(symbol.resolveAssetCallback, null);
     }
+
+    private _whitelist: {protocol: string; hostname: string; insecure: boolean}[] = [];
```

Comment:
Please give this a more descriptive name. Something like `crossDomainSWFLoadingWhitelist`.